src/helpers/CryptoHelper.ts

```diff
@@ -0,0 +1,253 @@
+      mac: buf.toString('hex', 49, 81),
+      ciphertext: buf.toString('hex', 81, buf.length),
+    };
+    // decompress publicKey
+    // encrypted.ephemPublicKey = new Buffer.from(publicKeyConvert(Web3Helper.getUint8ArrayFromHexStr(encrypted.ephemPublicKey), true)).toString('hex')
+    const input: Uint8Array = Uint8Array.from(new Buffer(encrypted.ephemPublicKey, 'hex'));
+    const keyConvert: any = publicKeyConvert(input, false);
+    // console.log("[DECRYPTION] Coverted key: " + keyConvert);
+
+    const keyConvertBuffer: Buffer = new Buffer(keyConvert);
+    // console.log("[DECRYPTION] Coverted key in buffer : " + keyConvertBuffer);
+    // console.log(keyConvertBuffer);
+
+    //console.log(keyConvert);
+    const uncompressedKey: string = keyConvertBuffer.toString('hex');
+    // console.log("[DECRYPTION] Uncompressed key in buffer : ");
+    // console.log(uncompressedKey);
+
+    encrypted.ephemPublicKey = uncompressedKey;
+    const twoStripped: string = privateKey.substring(2);
+
+    const encryptedBuffer: {
+      iv: Buffer;
+      ephemPublicKey: Buffer;
+      ciphertext: Buffer;
+      mac: Buffer;
+    } = {
+      iv: new Buffer(encrypted.iv, 'hex'),
+      ephemPublicKey: new Buffer(encrypted.ephemPublicKey, 'hex'),
+      ciphertext: new Buffer(encrypted.ciphertext, 'hex'),
+      mac: new Buffer(encrypted.mac, 'hex'),
+    };
+
+    return decrypt(new Buffer(twoStripped, 'hex'), encryptedBuffer).then((decryptedBuffer) =>
+      decryptedBuffer.toString()
+    );
+  },
+  // Testing of Encryption and Decryption from Public to Private key
+  encryptionDecryptionPublicToPrivateTest: async function (privateKey: string) {
+    const startTime: Date = new Date();
+    console.log('[ENCRYPTION / DECRYPTION TEST STARTED] - ' + startTime);
+
+    const publicKey: string = EthCrypto.publicKeyByPrivateKey(privateKey);
+    const compressedKey: string = EthCrypto.publicKey.compress(publicKey); // is String
+    //console.log(compressedKey);
+
+    // const bytesCompKey = Uint8Array.from(compressedKey);
+    //console.log(bytesCompKey);
+
+    const msgToEncrypt: string = 'PartialStringAS';
+    const msg: any = await this.encryptWithPublicKey(msgToEncrypt, compressedKey);
+    console.log('Encryped Message With compressed public key:' + msg);
+
+    const encryptionTime: number = new Date().getTime() - startTime.getTime();
+    console.log('[ENCRYPTION / DECRYPTION ENCRYPTION DONE] - ' + encryptionTime / 1000 + ' secs');
+
+    // Decrypt this message
+    const decryptMsg: string = await this.decryptWithPrivateKey(msg, privateKey);
+    console.log("[ENCRYPTION / DECRYPTION DECRYPTED MESSAGE] - '" + decryptMsg + "'");
+
+    const decryptionTime: number = new Date().getTime() - startTime.getTime() - encryptionTime;
+    console.log('[ENCRYPTION / DECRYPTION DECRYPTION DONE] - ' + decryptionTime / 1000 + ' secs');
+  },
+  // To output messge payload if required
+  outputMsgPayload: async function (
+    secret: string,
```

Comment:
wherever there are types with more than three properties, convert it into a type object and keep the common ones in the type folder.

Reply:
Also fix the eslint error that comes up when we load the dapp

Reply:
Made the suggested changes, didn't find any types used in different files so not moved any types to types folder.